kubernetes/base/config/__init__.py

```diff
@@ -33,7 +33,10 @@ def load_config(**kwargs):
     can be passed to either load_kube_config or
     load_incluster_config functions.
     """
-    if "kube_config_path" in kwargs.keys() or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
+    if "kube_config_path" in kwargs.keys():
+        kwargs["config_file"] = kwargs.pop("kube_config_path", None)
```

Comment:
if both `kube_config_path` and `config_file` are set, we should keep the value of `config_file`

Reply:
I added a check for the `config_file` value to ensure that it is used first